### PR TITLE
Improvement/site settings

### PR DESCRIPTION
**Pull request by santostiago**
The site creation feature.
As soon as it's merged, I'll create a branch to do the styling.

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -27,6 +27,7 @@
 // = require_tree ./templates
 // = require_tree ./views/shared
 // = require_tree ./views/front
+// = require_tree ./views/admin
```
Comment (clementprdhomme) [inline app/assets/javascripts/application.js]: You shouldn't do that. Why do you need to include admin's views here?

File: app/assets/javascripts/dispatchers/adminDispatcher.js [hunk 2]
```diff
@@ -4,7 +4,8 @@
   var Dispatcher = Backbone.Router.extend({
     routes: {
       '(/)': 'Index',
-      'site_settings/:id(/)': 'Settings'
+      'site_steps/settings(/)': 'Settings',
+      'site_steps/name(/)': 'Urls'
```
Comment (clementprdhomme) [inline app/assets/javascripts/dispatchers/adminDispatcher.js]: Preferably the name of the router could match the name of the route.

File: app/assets/javascripts/views/admin/urlsView.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+((function (App) {
+  'use strict';
+
+  var Collection = Backbone.Collection.extend({
```
Comment (clementprdhomme) [inline app/assets/javascripts/views/admin/urlsView.js]: This is not needed anymore.

File: app/assets/javascripts/views/admin/urlsView.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    // url: 'flag_colors.json'
+  });
+
+  App.View.UrlsView = Backbone.View.extend({
+    className: 'c-urls',
+    template: HandlebarsTemplates['admin/urls'],
+    collection: new Collection(
```
Comment (clementprdhomme) [inline app/assets/javascripts/views/admin/urlsView.js]: Don't forget to update this line:
```javascript
new Backbone.Collection(
```

File: app/assets/javascripts/views/admin/urlsView.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    },
+
+    _canRemoveUrl: function () {
+      return (this.collection.length > 0);
+    },
+
+    render: function () {
+      this.$el.html(this.template({
+        urls: this.collection.toJSON(),
+        inputId: (window.gon && gon.urlControllerId) ? gon.urlControllerId : '',
```
Comment (clementprdhomme) [inline app/assets/javascripts/views/admin/urlsView.js]: You could write these two lines like this:
```javascript
{
  inputId: window.gon && gon.urlControllerId || '',
  inputName: window.gon && gon.urlControllerName || ''
}
```

File: app/assets/javascripts/dispatchers/adminDispatcher.js (hunk 2 above)
Comment (santostiago) [inline app/assets/javascripts/dispatchers/adminDispatcher.js]: Ok.

**Pull request closed without merging**